homeassistant/components/media_player/sonos.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, hass, player):
         self.hass = hass
         self.volume_increment = 5
         self._player = player
+        self._name = None
```

Comment:
Default behavior at entity_id creation is to check for an id clash. If there is one an index will be added to the name. This index will be incremented at each creation and clash.